src/Database/Log/LoggingStatement.php

```diff
@@ -75,6 +77,21 @@ public function execute(?array $params = null): bool
             $result = parent::execute($params);
             $this->loggedQuery->took = (int)round((microtime(true) - $this->startTime) * 1000, 0);
         } catch (Exception $e) {
+            $this->loggedQuery->error = $e;
+            $this->_log();
+
+            if (Configure::read('Error.wrapStatementException', false) === true) {
```

Comment:
I think this name should be clearer.

convertSomethingToSomething?

Reply:
perhaps wrapStatementExceptionsWithDatabaseException ? I think the wrap terminology is more accurate because we also including the original.

Reply:
All exceptions can contain the previous one. I think convert conveys what is actually happening better.

Reply:
Maybe it could also be shorter with `convertStatementToDatabaseException` ?